### PR TITLE
DCOS-51242 - Confluent kafka 2.5.0 4.1.2

### PR DESCRIPTION
**Pull request by rathohit311356**
## Description
https://jira.mesosphere.com/browse/DCOS-51242

## Urgency
- [x] High

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+menuWeight: 10
+model: /services/confluent-kafka/data.yml
+render: mustache
+---
+# Release Notes for Confluent Kafka Service version 2.5.0-4.1.2
+
+## Updates
+
+- Update to Confluent Kafka version 4.1.2
+- Update dcos-coommons sdk version to 0.55.2
```
Comment (wavesoft) [inline pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md]: Can you also add a link that points to the upstream changelog? For the user's reference:
https://docs.confluent.io/4.1.2/release-notes.html

File: pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ render: mustache
 ## Updates
 
 - Update to Confluent Kafka version 4.1.2
-- Upgrade to sdk 0.55.2
+- Update dcos-coommons sdk version to 0.55.2
+- Update Zookeeper Client version to 3.4.13
```
Comment (wavesoft) [inline pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md]: Can you also a note that we are switching to file-based secrets here?

File: pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md (hunk 1 above)
Comment (rathohit311356) [reply pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md]: Done

File: pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md (hunk 2 above)
Comment (rathohit311356) [reply pages/services/confluent-kafka/2.5.0-4.1.2/release-notes/index.md]: Done

**Comment by rathohit311356** [comment]
Added Changes for Rlimit as well

**Pull request merged**